### ISSUE TITLE
Set a label for checkboxes in workflow extraction view

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#17194]
This fails accessibility best practices, by giving no other click targets than the relatively small checkbox. If the entire block could be the Label it would be friendlier.

![image](https://github.com/galaxyproject/galaxy/assets/458683/2c73f1c6-c4a4-4363-9e6c-a8992298fdb4)
